demo/src/javascripts/components/home.js

```diff
@@ -1,16 +1,14 @@
+/* global window */
```

Comment:
Hum I would rather declare only one, the less the better. Do you have a strong opinion on this?